app/views/personal_details/create.html.erb

```diff
@@ -1,18 +1,20 @@
 <span class="govuk-caption-xl"><%= t('application_form.personal_details_section.heading') %></span>
 <h1 class="govuk-heading-xl"><%= t('application_form.review_answers') %></h1>
 
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <dl class="govuk-summary-list govuk-!-font-size-16 govuk-summary-list--no-border">
-      <%= render partial: 'shared/description_list_item', locals: { field: :title, value: @personal_details.title } %>
-      <%= render partial: 'shared/description_list_item', locals: { field: :first_name, value: @personal_details.first_name } %>
-      <%= render partial: 'shared/description_list_item', locals: { field: :last_name, value: @personal_details.last_name } %>
-      <%= render partial: 'shared/description_list_item', locals: { field: :preferred_name, value: @personal_details.preferred_name } %>
-      <%= render partial: 'shared/description_list_item', locals: { field: :date_of_birth, value: @personal_details.date_of_birth.to_s(:govuk_human_readable) } %>
-      <%= render partial: 'shared/description_list_item', locals: { field: :nationality, value: @personal_details.nationality } %>
-    </dl>
+<% if @personal_details.valid? %>
```

Comment:
I think ought to have deleted this view in my branch as it's now never rendered — one for the rebase 🏗

Reply:
Fixed it via funky rebasing :tada: